### DISCUSSION TITLE
Friendly URLs. how to remove the /s after domain?

### DISCUSSION THREAD

**Discussion by axrusar** [nextcloud/all-in-one#1037] [Questions] [answered]
I am running AIO 24.0.3
I installed the Configurable Share Links app and i am able to create friendly URLs for my shared files, but there is an "/s" between the domain and the custom name. Ex:

`www.mydomain.com/s/MyCustomName`

I read some external articles about editing, .htaccess, or /var/www/html/nextcloud/config/config.php, 
but did not find anything with the docker option like AIO is using... 
I would like to remove the /s in between so the shares would look like:

`www.mydomain.com/MyCustomName`

Before i mess the current setup, would you guys point me in the right direction to accomplish this?

Thank you!!

**Comment by szaimen** [comment, marked as answer]
Hi, if AIO is running behind a reverse proxy, you may be able to do implement this by adding a custom entry to your reverse proxy config. However I would not recommend to do this as you may break things with that.

**Comment by axrusar** [comment]
Thank you szaimen.
My NC server is just behind the router, i am not using any proxies.
So you mean at least i am using a reverse proxy, then it is not recommended to alter the configuration on the NC server itself?

**Reply by szaimen** [reply]
> So you mean at least i am using a reverse proxy, then it is not recommended to alter the configuration on the NC server itself?

If you add a reverse proxy in between your router and AIO, you may be able to add a custom entry to your reverse proxy that allows you to do what you asked for. However I would not recommend adding this custom entry.
